homeworks/src/ltretykhlib/module3/Hackerrank8.java

```diff
@@ -0,0 +1,32 @@
+package ltretykhlib.module3;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.Set;
+
+/**
+ * Created by Tretykhlib on 20.04.2017.
+ */
+public class Hackerrank8 {
+    public static void main(String[] argh) {
+        Map<String, String> map = new HashMap<String, String>();
+        Scanner in = new Scanner(System.in);
+        int n = in.nextInt();
+        for (int i = 0; i < n; i++) {
+            String name = in.next();
+            int phone = in.nextInt();
+            map.put(name, Integer.toString(phone));
```

Comment:
Better to declare values to be of an Integer type in the map  ( Map<String,Integer>)